RadarSDK/RadarAPIClient.m

```diff
@@ -90,22 +105,20 @@ - (void)getConfig {
                                 sleep:NO
                            logPayload:YES
                     completionHandler:^(RadarStatus status, NSDictionary *_Nullable res) {
-                        if (!res) {
-                            return;
-                        }
-        
-                        [Radar flushLogs];
+        [Radar flushLogs];
```

Comment:
Is the indentation on these correct? Looks different than before

Reply:
Good call. I re-indented it using Xcode's ctrl-I, and Xcode changed its indentation rules at some point in the last couple of releases.

Reply:
we gotta fix our clang format tool usage

Reply:
If by "fix" you mean "don't use," then 👍. 😄